docs/fundamentals/systems/overlays/use-cases.md

```diff
@@ -132,14 +132,12 @@ render() {
 or declaratively in your template with the `.config` property
 
 ```html
-<demo-el-using-overlaymixin .config=${{ ...withModalDialogConfig() }}>
+<demo-el-using-overlaymixin .config="${{" ...withModalDialogConfig() }}>
```

Comment:
This one goes wrong

Reply:
Hehe this is funny. Prettier tries to parse these lit templates as HTML which of course fails. I changed lit snippets to JSX which seems to fix it but it's a shame that prettier doesn't seem to have Lit template support.